### ISSUE TITLE
"Tip: Based on detected gems..." is appended to file when --stdin option is used e.g. in editor integration

### ISSUE THREAD

**Issue by henrahmagix** [rubocop/rubocop#9617]
This makes the `-s|--stdin` option unviable for editor integration.

I'm using VS Code with the Ruby extension, and it uses the output to replace the file contents when format-on-save is turned on. [Here's the extension](https://github.com/rubyide/vscode-ruby/blob/7b5b6025caa2ebc140192e9add094aa34e7cec2d/packages/language-server-ruby/src/formatters/RuboCop.ts) that produces the `rubocop ...` line.

My VS Code settings for this:
```json
"ruby.useLanguageServer": true,
"editor.formatOnSave": true,
"ruby.format": "rubocop",
```

--------

## Expected behavior

Rubocop detects when it's outputting to stdout and [skips SuggestExtensions](https://github.com/rubocop/rubocop/blob/255692c888c6187e4e0564e4f46e37cddce6a8e4/lib/rubocop/cli/command/suggest_extensions.rb#L40-L48). Or it prints it before the output delimiter so scripts ignore it.

## Actual behavior

The suggested extensions copy gets appended to the file that was formatted by rubocop in my code editor.

## Steps to reproduce the problem

Given some gems that produce suggested extensions, e.g. `minitest`, and a file `/path/to/file.rb` that contains some ruby code:
```rb
module RubocopBugEmptyFile; end
```

This is ok as it's human-usage, not scripted:
```bash
$: rubocop /path/to/file.rb
Inspecting 1 file
.

1 file inspected, no offenses detected

Tip: Based on detected gems, the following RuboCop extension libraries might be helpful:
  * rubocop-minitest (http://github.com/rubocop-hq/rubocop-minitest)

You can opt out of this message by adding the following to your config (see https://docs.rubocop.org/rubocop/extensions.html#extension-suggestions for more options):
  AllCops:
    SuggestExtensions: false
```

This is incorrect as it's meant to be used by scripts for editor integration:
```bash
$: rubocop -s '/path/to/file.rb' -a < /path/to/file.rb
Inspecting 1 file
.

1 file inspected, no offenses detected
====================
module Empty; end

Tip: Based on detected gems, the following RuboCop extension libraries might be helpful:
  * rubocop-minitest (http://github.com/rubocop-hq/rubocop-minitest)

You can opt out of this message by adding the following to your config (see https://docs.rubocop.org/rubocop/extensions.html#extension-suggestions for more options):
  AllCops:
    SuggestExtensions: false
```

As far as I can tell, based on [VS Code's Ruby extension known "output delimiter"](https://github.com/rubyide/vscode-ruby/blob/7b5b6025caa2ebc140192e9add094aa34e7cec2d/packages/language-server-ruby/src/formatters/RuboCop.ts#L6), I think it should be this:
```bash
$: rubocop -s '/path/to/file.rb' -a < /path/to/file.rb
Inspecting 1 file
.

1 file inspected, no offenses detected
====================
module Empty; end
```

## RuboCop version

Include the output of `rubocop -V` or `bundle exec rubocop -V` if using Bundler.
If you see extension cop versions (e.g. `rubocop-performance`, `rubocop-rspec`, and others)
output by `rubocop -V`, include them as well. Here's an example:

```
$ [bundle exec] rubocop -V
1.11.0 (using Parser 3.0.0.0, rubocop-ast 1.4.1, running on ruby 2.6.6 x86_64-darwin19)
  - rubocop-rails 2.9.1
  - rubocop-rspec 2.1.0
```

**Comment by henrahmagix** [comment]
Aw wow, that was super quick, thanks so much @dvandersluis!

Could you kindly comment here when it's released, so I get a notification please? Or if you know roughly when it might get released so I can set a reminder? Thanks a bunch!

**Comment by dvandersluis** [comment]
Thanks for the issue @henrahmagix!

I'm not sure when the next release is planned for but you can have github notify you about releases specifically in the Watch menu for the repo.
